### DISCUSSION TITLE
Настройщик Мышления или Мыслей?

### DISCUSSION THREAD

**Discussion by tigran123** [tigran123/ud#2] [Идеи и вопросы]
Настройщик Мышления или Мыслей?
 
В 101:1.3 сказано, что Божественный дух контактирует со смертным в сфере одухотворённого _**мышления**_, но именно _**мысли**_, а не чувства или эмоции ведут нас к Богу.

Так как лучше перевести английское Thought Adjuster -- Настройщик Мышления или Мыслей?

**Comment by irinachernov** [comment]
Мне больше нравится: Настройщик Мыслей. Ведь к Отцу мы в основном обращаемся в мыслях.

**Reply by tigran123** [reply]
И Отец отвечает осторожным и незаметным вмешательством в наш поток мыслей.

**Comment by irinachernov** [comment]
On Thu, Jan 7, 2021 at 9:35 AM Tigran Aivazian <notifications@github.com>
wrote:

> И Отец отвечает осторожным и незаметным вмешательством в наш поток мыслей.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/tigran123/ud/discussions/2#discussioncomment-267089>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARZMOFVTW2ZMOKOSCTPT7DDSYXBB5ANCNFSM4VXV45XQ>
> .
>
Да, он делает это так корректно!  :)--
*Sincerely,*
*Irina Chernov*

**Comment by tigran123** [comment]
Сказано -- сделано: https://github.com/tigran123/ud/commit/de67efe687c46120f19c9560dfa26b3e41ef62ea

**Reply by irinachernov** [reply]
Здорово, спасибо! :)
